### ISSUE TITLE
[Enhancement]Rest api

### ISSUE THREAD

**Issue by ghost** [sqlilabs/WorkshopManager#76]
Dit would ne great to have  a full API tout browser workshops details

**Comment by ychartois** [comment]
Of course... if it was design to be a REST api.... I know really bad design... but it was few years ago now.... if you want to do something better like 100% REST and AngularJS I'm in... but can't do it alone that time...
